src/agent/driver-test-runner.js

```diff
@@ -119,21 +119,29 @@ export class DriverTestRunner {
         if (!desiredResponse) {
           throw new Error(`Unknown command settings for NVDA "${settings}"`);
         }
-        const MODE_SWITCH_SPEECH_TIMEOUT = 500;
-        let speechResponse;
+        // try to keep this short because it runs up to twice for every test
+        // have seen occasional timeout/misses at 500ms
+        const MODE_SWITCH_SPEECH_TIMEOUT = 750;
```

Comment:
This is fine for a quick fix, but in general, a variable should be controlled from the context upon which it depends. What that means here is this value should be specified command-line argument. @ccanash Can you create a Task/Issue to track that change?

Reply:
[Here's a GitHub Issue to track that work.](https://github.com/w3c/aria-at-automation-harness/issues/49)